usr/bin/hwctl

```diff
@@ -198,6 +198,15 @@ function storage {
 		fi
 		echo "INFO: Reloading system partition information"
 		partprobe 2> /dev/null
+
+		partition_name=$(basename ${partition_to_format})
+		mount_point="/run/media/${partition_name}.ext4"
+		sleep 1 # give time for automount
```

Comment:
I would say that we can use `udevadm settle` here but the automount is handled by a service can it is run with the argument `--no-block` so it exits completely. A `sleep` here is fine then.

https://github.com/Ferk/udev-media-automount/blob/7f7aa156652ea4172070b35e67d6507edb15ec35/media-automount.rules#L12

Reply:
I think we can use UDisks2 here to mount it. 

```
busctl --system call org.freedesktop.UDisks2 /org/freedesktop/UDisks2/block_devices/sda1 org.freedesktop.UDisks2.Filesystem Mount a{sv} 1 "as-user" s "playtron"
```

playtron should be the user name, I believe that's the default. 

sda1 should be the name of the partition under /dev. 

This should mount it the same way that automount does

Reply:
NOTE: the sleep is here "just in case". It didn't seem to actually be needed, but I thought it might help under some circumstances.

So, udisks does not mount the disks in the same way/location as udevadm.
With udisks, you get `/run/media/playtron/<device uuid>` while udevadm mounts to `/run/media/<device label or name>.<filesystem type>`

Reply:
I see. In Fedora the udisks behaviour is the default